spec/features/user_can_log_in_and_out_spec.rb

```diff
@@ -4,14 +4,14 @@
     scenario "logs into users account" do
       user = create(:user)
 
-      visit "/"
+      visit root_path
       click_on "Login"
-      
+
       expect(current_path).to eq(login_path)
 
       fill_in("session[email]", with: user.email)
       fill_in("session[password]", with: "password")
-      click_on "Login"
+      click_on "Log in"
```

Comment:
Did you change this to "log in" so capybara could find it?  I'm worried it will look strange because "login" is pretty much standard.  We should be able to differentiate the login link from the button based on the html type.  I can help with this, I've done it before.